docs/user-guide/service-accounts.md

```diff
@@ -120,14 +120,41 @@ $ kubectl delete serviceaccount/build-robot
 ```
 
 <!-- TODO: describe how to create a pod with no Service Account. -->
+Note that if a pod does not have a `ServiceAccount` set, the `ServiceAccount` will be set to `default`.
 
 ## Adding Secrets to a service account.
 
-TODO: Test and explain how to use additional non-K8s secrets with an existing service account.
+Suppose we have an existing service account named "build-robot" as mentioned above, and we create
+a new secret manually.
 
-TODO explain:
-  - The token goes to: "/var/run/secrets/kubernetes.io/serviceaccount/$WHATFILENAME"
+```console
+$ cat > /tmp/build-robot-secret.yaml <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: build-robot-secret
+  annotations: 
+    kubernetes.io/service-account.name: build-robot
+type: kubernetes.io/service-account-token
+EOF
+$ kubectl create -f /tmp/build-robot-secret.yaml
+secrets/build-robot-secret
+```
 
+Now you can confirm that the newly built secret is populated with an API token for the "build-robot" service account.
+
+```console
+kubectl describe secrets/build-robot-secret 
+Name:   build-robot-secret
+Namespace:  default
+Labels:   <none>
+Annotations:  kubernetes.io/service-account.name=build-robot,kubernetes.io/service-account.uid=870ef2a5-35cf-11e5-8d06-005056b45392
+
+Type: kubernetes.io/service-account-token
+
+Data
+====
+```
```

Comment:
there should be a `token` key in the `Data` section... just elide the content for this example